pywatts/wrapper/sm_time_series_model_wrapper.py

```diff
@@ -0,0 +1,170 @@
+import inspect
+from typing import Dict, Type
+
+import numpy as np
+import xarray as xr
+from statsmodels.iolib import load_pickle
+from statsmodels.tsa.base.tsa_model import TimeSeriesModel
+
+from pywatts.core.filemanager import FileManager
+from pywatts.utils._xarray_time_series_utils import _get_time_indeces, numpy_to_xarray
+from pywatts.wrapper.base_wrapper import BaseWrapper
+
+
+class SmTimeSeriesModelWrapper(BaseWrapper):
+    """
+    Wrapper for statsmodels modules.
+
+    :param module: The statsmodels module to wrap. Not this module should not be initialised.
+    :param name: The name of the module
+    :type name: str
+    :param module_kwargs: The module keyword arguments necessary for creating the statsmodel module
+    :type module_kwargs: dict
+    :param fit_kwargs: The optional fit keyword arguments for fitting the model
+    :type fit_kwargs: dict
+    :param predict_kwargs: The optional predict keyword arguments for predicting with the model (except start and end)
+    :type predict_kwargs: dict
+    """
+
+    def __init__(self, module: Type[TimeSeriesModel], name: str = None, module_kwargs=None,
+                 fit_kwargs=None, predict_kwargs=None):
+        if name is None:
+            name = module.__name__
+        super().__init__(name)
+        if fit_kwargs is None:
+            fit_kwargs = {}
+        if predict_kwargs is None:
+            predict_kwargs = {}
+        self.module = module
+        if not module_kwargs:
+            module_kwargs = {}
+        self.module_kwargs = module_kwargs
+        self.fit_kwargs = fit_kwargs
+        self.predict_kwargs = predict_kwargs
+
+    def get_params(self) -> Dict[str, object]:
+        """
+        Returns the parameters of the statsmodels module.
+
+        :return: A dict containing the module keyword arguments, the fit keyword arguments, the predict keyword
+        arguments and the fitted model parameters
+        :rtype: Dict
+        """
+        return {
+            "module_kwargs": self.module_kwargs,
+            "fit_kwargs": self.fit_kwargs,
+            "predict_kwargs": self.predict_kwargs,
+        }
```

Comment:
Shouldn't the self.module attribute be part of the get_params return dict?

Reply:
Currently no wrapper returns the wrapped module. Consequently also the statsmodel wrapper should not do it.